phantom-dsl/src/test/scala/com/outworkers/phantom/jdk8/tables/PrimitivesJdk8.scala

```diff
@@ -30,6 +30,8 @@ case class Jdk8Row(
   localDateTime: LocalDateTime
 )
 
+import Bla._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be grouped together](https://app.codacy.com/manual/flavian/phantom/pullRequest?prid=3695041)